examples/template.html

```diff
@@ -124,6 +124,10 @@
             animate();
           });
 
+          $(document).ready(function(){
```

Comment:
Just tested that out.  If we move everything into document.ready(), it breaks.
```javascript
          $(document).ready(function(){
             init();
             animate();
             ec.resize(window.innerWidth, window.innerHeight);
          });
```
Here's something funny - if I have two declarations of document.ready(), it works again
```javascript
          $(document).ready(function(){
             init();
             animate();
          });

          $(document).ready(function(){
             ec.resize(window.innerWidth, window.innerHeight);
          });
```